gitpod-samples/template-ruby-on-rails-postgres
# fix: Ruby PATHs not being set correctly

## Description


## Related Issue(s)

Fixes [this on discord](https://canary.discord.com/channels/816244985187008514/933654077492035614/934429588623347763)

## How to test

- Open this PR in Gitpod
- Run `rails console` and see if it works.

## Release Notes

```release-note
```

## Documentation
